packages/beacon-node/src/metrics/metrics/lodestar.ts

```diff
@@ -716,6 +717,27 @@ export function createLodestarMetrics(
         buckets: [0.05, 0.1, 0.2, 0.5, 1, 1.5, 2, 4],
       }),
     },
+    recoverDataColumnSidecars: {
+      secFromSlot: register.histogram({
+        name: "lodestar_recover_data_column_sidecar_recover_timer_from_slot",
+        help: "Time elapsed between block slot time and the time data column sidecar recovered",
+        buckets: [2, 4, 6, 8, 10, 12],
+      }),
+      recoverTime: register.histogram({
+        name: "lodestar_recover_data_column_sidecar_recover_time_seconds",
+        help: "Time elapsed to recover data column sidecar",
+        buckets: [0.5, 1, 2, 3, 4],
```

Comment:
With rust-kzg being much faster how do you feel about adjusting the buckets a bit?
```suggestion
        buckets: [0.4, 0.6, 0.8, 1.0, 1.2],
```

Reply:
yeah that old metric went with `c-kzg`
new buckets look good to me with 20 blobs, can change it in the future